### PR TITLE
BDOG-2362 Libraries - Show all versions dependencies

### PR DESCRIPTION
**Pull request by RikSherman**

### CODE REVIEW

**Review by christopherjturner — APPROVED** [review]

File: app/views/LibraryInfoPage.scala.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 )
 
 @(library                : GitRepository,
-  repositoryModules      : Option[RepositoryModules],
+  metaInfo               : Seq[RepositoryModules],
```
Comment (colin-lamed) [inline app/views/LibraryInfoPage.scala.html:32]: The name `metaInfo` is a bit ambiguous (especially when you see things like `metaInfo.headOption`), I'd be tempted to leave it as `repositoryModules` (even if there is a lack of double plural)?

File: app/views/LibraryInfoPage.scala.html [hunk 2]
```diff
@@ -51,7 +51,7 @@ <h1>
   </header>
 
   @partials.leak_detection_banner(linkToLeakDetection)
-  @defining(repositoryModules.fold(Seq.empty[Dependency])(_.allDependencies)){ dependencies =>
+  @defining(metaInfo.headOption.fold(Seq.empty[Dependency])(_.allDependencies)){ dependencies =>
```
Comment (colin-lamed) [inline app/views/LibraryInfoPage.scala.html:54]: can we use `@defining(metaInfo.headOption){ latestVersion =>` to explain what's going on, and avoid multiple calls to `metaInfo.headOption`?

File: app/views/LibraryInfoPage.scala.html [hunk 3]
```diff
@@ -78,10 +78,39 @@ <h1>
       <div class="col-md-6">
         @partials.build(library, hasBranchProtectionAuth, commenterReport)
       </div>
+
+        <div class="col-md-6">
+        @partials.dependency_legend(true)
```
Comment (colin-lamed) [inline app/views/LibraryInfoPage.scala.html:83]: what is `true`? can we use a named param?

File: app/views/partials/ModulesPartial.scala.html [hunk 4]
```diff
@@ -21,7 +21,9 @@
 )
 
 @(
-  repositoryModules: RepositoryModules
+  repositoryModules : RepositoryModules,
+  isLatest          : Boolean         = false,
```
Comment (colin-lamed) [inline app/views/partials/ModulesPartial.scala.html:25]: wonder if naming `isLatest` as `displayBuildInfo` will decouple it from domain knowledge, and make it easier to understand why it's being asked for?

**Pull request merged**